bin/deploy-hobby

```diff
@@ -100,10 +124,12 @@ curl -L --header "Content-Type: application/json" -d '{
 }' https://app.posthog.com/batch/
```

Comment:
Totally off topic, but is this capturing anything? 
<img width="2110" alt="image" src="https://user-images.githubusercontent.com/53387/154653847-5e0dff71-ec9f-4534-a061-6724ca7bed0e.png">

Reply:
it doesn't - need to fix in another PR. Very broken